### ISSUE TITLE
Unable to connect with Docker Extension using clusters with external authenticators

### ISSUE THREAD

**Issue by anafilipadealmeida** [datawire/telepresence-docker-extension-issues#12]
The extension runs TP inside a container, so it's not able to use external programs on the user's machine. (eg. GKE or AWS authenticator)

**Comment by nicks** [comment]
when I run Telepresence in a container, we usually:
1) copy the aws-iam-authenticator binary into the container
2) mount ~/.aws to /root/aws

and that usually gets it

(gke can be a bit messier because gcloud is in python and has python versioning hell, but aws-iam-authenticator is a static binary)

**Comment by knlambert** [comment]
It will be fixed with the [upcoming release](https://github.com/telepresenceio/telepresence/pull/3055) since we'll use the cli directly instead of running telepresence in a container.

We fixed it by adding a service running on the user machine taking care of interacting with these binaries on the host machine, while the binaries running in the container are just stubs.

**Comment by khussey** [comment]
The fix for this issue is now available in version 1.0.0 of the Telepresence Docker Extension.

**Comment by nicks** [comment]
for what it's work, this still doesn't work for me, i get this error:

```
connector.Connect: initial cluster check failed: Get "https://B28838B96CE877B06624F369714DECF5.gr7.us-east-1.eks.amazonaws.com/version": getting credentials: exec: executable authenticator failed with exit code 1
```

**Comment by khussey** [comment]
> for what it's work, this still doesn't work for me, i get this error:
> 
> ```
> connector.Connect: initial cluster check failed: Get "https://B28838B96CE877B06624F369714DECF5.gr7.us-east-1.eks.amazonaws.com/version": getting credentials: exec: executable authenticator failed with exit code 1
> ```

This should be addressed with version 1.0.1 of the extension and Telepresence 2.12.1, assuming you are specifying an absolute path to your authenticator in your kubeconfig as advised at https://www.getambassador.io/docs/telepresence/latest/install/cloud#eks-authentication-plugin.
